src/vcf/record.cpp

```diff
@@ -258,24 +258,24 @@ namespace ebi
                         check_field_cardinality(field.second, values, key_values[NUMBER]);
                         check_field_type(values, key_values[TYPE]);
                     } catch (std::shared_ptr<Error> ex) {
-                        std::string message = "INFO " + key_values[ID] + "=" + field.second
-                                + " does not match the meta" + ex->message;
-                        throw new InfoBodyError{line, message, ErrorFix::IRRECOVERABLE_VALUE, key_values[ID]};
+                        std::string message = "INFO " + key_values[ID] + " does not match the meta" + ex->message;
+                        throw new InfoBodyError{line, message, key_values[ID] + "=" + field.second,
```

Comment:
There's another place were you can do this, in record_cache. I tested the summary report and looks great, but not so much for duplicated variants:
> Duplicated variant 0:0:A>G found in lines 34 and 35. This occurs 2 time(s), first time in line 34
Duplicated variant 0:0:A>G found in lines 34 and 36. This occurs 1 time(s), first time in line 36
Duplicated variant 0:0:A>G found in lines 34 and 37. This occurs 1 time(s), first time in line 37
Duplicated variant 0:0:A>G found in lines 34 and 38. This occurs 1 time(s), first time in line 38

As you see, we are including the second appearance (35, 36, 37, 38) in the main message, which ends up in different entries in the map. Ideally this would show as:

> Duplicated variant 0:0:A>G found. This occurs 5 time(s), first time in line 34

the debugulator is able to remove those, so there's no need to tell which lines it was found to the user (those can still be in the detailed message so that the full text report still has the lines).

Reply:
ohh sorry i forsgot it

Reply:
this is what the errors look like now in the full text report:
```
Line 34: Duplicated variant 0:0:A>G found.
Line 35: Duplicated variant 0:0:A>G found. It occurs in lines 34, 35
Line 36: Duplicated variant 0:0:A>G found. It occurs in lines 34, 36
Line 37: Duplicated variant 0:0:A>G found. It occurs in lines 34, 37
Line 38: Duplicated variant 0:0:A>G found. It occurs in lines 34, 38
```